src/awx_plugins/interfaces/_temporary_private_api.py

```diff
@@ -4,6 +4,13 @@
 The hope is that it will be refactored into something more standardized.
 """
 
+from collections.abc import Callable
+
+from awx_plugins.interfaces._temporary_private_credential_api import (  # noqa: WPS436, pylint: disable=C0301
```

Comment:
Urgh.. I missed this. Don't use cryptic numbers in pylint ignores. It's got proper human-readable names for rules...

Reply:
Anyway, the root cause can be fixed because the linting violation actually reveals a different problem — an absolute import. This should be using relative imports instead. And then, there wouldn't be a problem to suppress in the first place. We shouldn't hide such problems, but work to understand why they happen.

Reply:
If something needs to be ignored, there should be a code comment explaining why. Especially for uncommon flake8 rule codes. In case of pylint, they should be human-readable, which is often enough (but not always).